### DISCUSSION TITLE
CNC 3018 Pro Laser Problems

### DISCUSSION THREAD

**Discussion by locovaldes** [arkypita/LaserGRBL#1158] [Q&A]
Hi i have a 3018 Pro , and the laser was working good and now sometime burn and sometime nothing, or do like small draw no normal
![123](https://user-images.githubusercontent.com/73902303/98018132-2e2b7300-1dce-11eb-88c1-c57ae4bb1573.JPG)

**Comment by locovaldes** [comment]
first was working fine 
![123189056_401928250983547_3121162529269964463_n](https://user-images.githubusercontent.com/73902303/98018258-561ad680-1dce-11eb-9c2b-6daf30b072a6.jpg)
![123452324_373113303930923_8990102190788775060_n](https://user-images.githubusercontent.com/73902303/98018260-56b36d00-1dce-11eb-90f0-07a17a2886d5.jpg)

and now i try to do the 800 speed test and see what i got

**Comment by locovaldes** [comment]
first was and now i try to do the 800 speed test and see what i got

**Comment by locovaldes** [comment]
and sometime no burn nothing

**Comment by gmmanonymus111** [comment]
While importing your image/vector, did you set the correct `S-Min` and `S-Max` values?
While importing images, did you set the correct raster settings?

(NOTE: the values on the pictures are for reference, random values)
![image](https://user-images.githubusercontent.com/9658243/98026523-8cc31200-1e0b-11eb-9349-7223cbc5813e.png)
![image](https://user-images.githubusercontent.com/9658243/98026771-ec212200-1e0b-11eb-8746-abcb7e79bb38.png)

**Comment by locovaldes** [comment]
Hi if i use LaserGRBL where can i see that

**Comment by locovaldes** [comment]
i importing a G code

**Comment by locovaldes** [comment]
I did it this yesterday
![20201102_102414](https://user-images.githubusercontent.com/73902303/98048317-6c8b5700-1dfb-11eb-855c-53ed46598b67.jpg)

**Comment by locovaldes** [comment]
Right now not good
![20201103_173716](https://user-images.githubusercontent.com/73902303/98048344-790faf80-1dfb-11eb-9366-b7c564293197.jpg)

**Comment by gmmanonymus111** [comment]
How old is your machine? It might be your diode is dying.
It's also worth to check the wiring connection, there might be some electrical noise, preventing your laser driver to receive correct PWM signal. Check your power supply, and make sure it is at least 2A.

**Comment by locovaldes** [comment]
Is a new machineSent from my T-Mobile 4G LTE Device
-------- Original message --------From: Alex <notifications@github.com> Date: 11/4/20  1:36 PM  (GMT-05:00) To: arkypita/LaserGRBL <LaserGRBL@noreply.github.com> Cc: locovaldes <locovaldes@yahoo.com>, Author <author@noreply.github.com> Subject: Re: [arkypita/LaserGRBL] CNC 3018 Pro Laser Problems (#1158) 
How old is your machine? It might be your diode is dying.
It's also worth to check the wiring connection, there might be some electrical noise, preventing your laser driver to receive correct PWM signal. Check your power supply, and make sure it is at least 2A.

**Comment by BeatSlayer** [comment]
Are you focusing it?

**Comment by locovaldes** [comment]
Yes thanks I already resolve the problem.Sent from my T-Mobile 4G LTE Device
-------- Original message --------From: BeatSlayer <notifications@github.com> Date: 11/5/20  5:13 PM  (GMT-05:00) To: arkypita/LaserGRBL <LaserGRBL@noreply.github.com> Cc: locovaldes <locovaldes@yahoo.com>, Author <author@noreply.github.com> Subject: Re: [arkypita/LaserGRBL] CNC 3018 Pro Laser Problems (#1158) 
Are you focusing it?

**Reply by JamieMcJannett** [reply]
Can you please tell me what you did to resolve this as I am having the exact same issue.

**Reply by karen1202** [reply]
Can you please tell us what did you do? I have the same problem :( I'm worry! I have a lot of work to finish
